chips_wrappers/setup/make_args.py

```diff
@@ -3,11 +3,11 @@
 import shlex
 from copy import deepcopy
 
-SCALAR_DENSITY_CORRECTION = 2.6505481727808022
+SCALAR_DENSITY_CORRECTION = 2.08704581 # adjust normalization of density correction to match 10mK noise simulation
```

Comment:
the comment implies that we were expecting 10mK but Jack reckons the file had 16mK